packages/flutter/lib/src/material/autocomplete.dart

```diff
@@ -105,6 +117,23 @@ class Autocomplete<T extends Object> extends StatelessWidget {
   /// {@macro flutter.widgets.RawAutocomplete.initialValue}
   final TextEditingValue? initialValue;
 
+  /// If the options view overlay should be shown when the options returned from
+  /// [optionsBuilder] is empty.
+  ///
+  /// Set to true to display a custom options view on
+  /// first focus or to display a message if no options are found.
+  ///
+  /// If not provided, will only display an options view when
+  /// some options have been returned from [optionsViewBuilder].
```

Comment:
Nit: "some options" => "at least one option"

That is more clear to me, but I'm not sure, up to you.